CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_ad_reporting v1.13.0
+[PR #137](https://github.com/fivetran/dbt_ad_reporting/pull/137) includes the following updates:
+
+## Breaking Changes: Version Updates
+The following dependencies have been updated following upstream breaking changes. See the below release notes for more information.
+  - `dbt_apple_search_ads` ([v0.5.0](https://github.com/fivetran/dbt_apple_search_ads/releases/tag/v0.5.0))
+  - `dbt_microsoft_ads` ([v0.10.0](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.10.0))
+
+## Bug Fixes
+- Resolved a CLI Warning caused by the `metricflow_time_spine` model not having a properly documented YAML configuration.
+
+## Under The Hood
+- Updated `apple_search_ads` and `microsoft_ads` seed files to keep consistent with changes to seed files in the individual packages.
+- The `conversions` field for `apple_search_ads` now sources from `tap_installs`.
+- Removed the dependency on [calogica/dbt_date](https://github.com/calogica/dbt-date) as it is no longer actively maintained. To maintain functionality, the `get_base_dates` macro used in `metricflow_time_spine` semantic model has been replicated and housed within the `macros/fivetran_date_macros` [folder](https://github.com/fivetran/dbt_ad_reporting/tree/main/macros/fivetran_date_macros). All relevant macros have been prefixed with `fivetran_` to avoid potential naming conflicts.
```

Comment:
This should be moved to the breaking changes: version updates section

Reply:
I would also make a sub bullet stating that we have retained the variable name/usage of `dbt_date:time_zone` to make this transition easier for users. It will be relevant for them to know that although we are removing the dependency and renaming macros, this variable will still be used the same.